Observer/Admin/DashboardInstantiation.php

```diff
@@ -119,22 +119,26 @@ public function __construct(
     public function execute(Observer $observer) : void
     {
         try {
-            if ($this->config->isMerchantActive() || $this->registry->registry(self::ACCESS_TOKEN_SET_KEY)) {
+            $store = $this->storeManager->getStore();
+
+            // TODO use $storeId
+            if ($this->config->isMerchantActive(null) || $this->config->getAccessToken(null)) {
                 return;
             }
 
             $moduleData = $this->moduleList->getOne('NS8_Protect');
             $moduleVersion = $moduleData['setup_version'] ?? '';
             $storeEmail = $this->scopeConfig->getValue('trans_email/ident_sales/email') ?? '';
-            $storeUrl = rtrim($this->storeManager->getStore()->getBaseUrl(UrlInterface::URL_TYPE_WEB, true), '/');
+            $storeUrl = rtrim($store->getBaseUrl(UrlInterface::URL_TYPE_WEB, true), '/');
             $installRequestData = [
                 'storeUrl' => $storeUrl,
                 'email' => $storeEmail,
                 'moduleVersion' => $moduleVersion,
                 'platformVersion' => (string) $this->productMetadata->getVersion()
             ];
 
-            $installResult = InstallerClient::install('magento', $installRequestData);
+            // $installResult = InstallerClient::install('magento', $installRequestData);
+            $installResult = [ 'accessToken' => '8a066db0-9459-49ec-a388-2bff1897b1ac' ];
```

Comment:
Just checking, is this intended for the PR?

Reply:
good catch, no